versions.yaml

```diff
@@ -105,8 +105,7 @@ assets:
       snp:
         description: "VMM that uses KVM and supports AMD SEV-SNP"
         url: "https://github.com/AMDESE/qemu"
-        branch: "snp-v3"
-        commit: "ffa95097ee"
```

Comment:
We need this to be pinned to a commit for reproducibility. The old commit is fine, but the current tip of snp-v3 would also be fine: 3b6a2b6b74

Reply:
This:
```
tag: "3b6a2b6b7466f6dea53243900b7516c3f29027b7"
```
also seems to work